ansible/ansible-ui
# Initial E2E test for hub dashboard using galaxykit resources

This PR enables creating and cleaning up of Hub collections/namespaces for E2E tests using the `galaxykit` command line tool in Cypress.
This PR does not include `galaxykit`'s installation for CI and for now only enables running these tests locally.

Steps to test locally:
1. Install `galaxykit` locally
    - Clone repo https://github.com/ansible/galaxykit
    - `cd galaxykit`
    - `pip3 install -e .`
2. In the `ansible-ui` directory:
    - Check that you have the ansible binary installed correctly under the Python directory. Example: `/Users/vidyanambiar/Library/Python/3.9/bin`. Add this directory to your PATH variable: Example: 
    ```bash
    export PATH="/Users/vidyanambiar/Library/Python/3.9/bin:$PATH"
    source ~/.zshrc
    ```
